product_warranty/models/product_supplierinfo.py

```diff
@@ -81,22 +55,27 @@ def _compute_warranty_return_address(self):
                 "brand manufacturer. Doesn't necessarly mean that the "
                 "warranty to be applied is the one of the return partner "
                 "(ie: can be returned to the company and be under the "
-                "brand warranty")
+                "brand warranty)"
+    )
     return_instructions = fields.Many2one(
         'return.instruction',
         'Instructions',
         default=_get_default_instructions,
-        help="Instructions for product return")
+        help="Instructions for product return"
```

Comment:
I'm for tidying up the parenthesis like this, but could you also add trailing `,` after all of the last arguments as well? This allow for a clean diff on that line if we need to add things 😄 

```
help="Instructions for product return",
```